pkg/validationfile/loader_test.go

```diff
@@ -0,0 +1,138 @@
+package validationfile
+
+import (
+	"testing"
+
+	v0 "github.com/authzed/authzed-go/proto/authzed/api/v0"
+	"github.com/authzed/spicedb/internal/datastore/memdb"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPopulateFromFiles(t *testing.T) {
+	tests := []struct {
+		name      string
+		filePaths []string
+		want      []*v0.RelationTuple
+	}{
+		{
+			name:      "no comment",
+			filePaths: []string{"testdata/loader_no_comment.yaml"},
+			want: []*v0.RelationTuple{
+				{
```

Comment:
You can use `tuple.Parse` to make this more compact (not required, but makes it a bit more readable)